src/Payum/LaravelPackage/Model/Token.php

```diff
@@ -16,6 +16,11 @@ class Token extends Model implements TokenInterface
      * @var string
      */
     protected $primaryKey = 'hash';
+    
+    /**
+     * @var bool
+     */
+    public $incrementing = false;
```

Comment:
It is public on the parent class.  https://github.com/laravel/framework/blob/5.2/src/Illuminate/Database/Eloquent/Model.php#L70